content/en/agent/guide/docker-deprecation.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Docker Deprecation in Kubernetes
+kind: documentation
+---
+
+Kubernetes is deprecating Docker as a runtime starting after version 1.20, and some cloud providers have deprecated Docker in their images. 
+
+- AKS 1.19 [deprecated Docker and uses containerd by default][1].
+
+- GKE 1.19 [deprecated Docker and uses containerd by default, on new nodes][2].
+
+If you are running a version of Kubernetes where Docker has been deprecated, the Docker socket is no longer present, and the Docker check does not work. This means that you will have to enable either the [containerd][3] or the [CRI-O][4] check depending on the container runtime you are using. The container metrics collected from the new container runtime replace the Docker metrics.
+
+With version 7.27+ of the Datadog Agent, the Agent automatically detects the environment you are running, and you do not to make any configuration changes.
+
+**If you are using Agent < v7.26, then you will need to specify your container runtime socket path:**
+
+Note that you may need to update your existing monitors, dashboards and SLOs because metrics names will change—for example, from `docker.*` to `containerd.*`.
```

Comment:
```suggestion
**Note**: You may need to update your existing monitors, dashboards, and SLOs because metrics names will change—for example, from `docker.*` to `containerd.*`.
```